examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+# 🏷 Continuous Annotation with Label Studio
+
+Data annotation / labeling is a core part of MLOps that is frequently left out
+of the conversation. ZenML offers a way to build continuous annotation (combining
+training and annotation into a loop) with Label Studio. This uses a combination
+of user-defined steps as well as some built-in steps that ZenML provides.
+
+## Basic guide to running this example
+
+This example requires a few more steps than usual, given that it requires a
+cloud artifact store. Instructions for Azure, AWS S3 and GCP can be found in the
+following guide.
+
+## 🗺 Overview
+
+NOTE: This example currently only runs with a cloud artifact store. [This guide
+by Label Studio](https://labelstud.io/guide/storage.html) contains more details
+if you have trouble understanding the instructions below.
+
+For a full video walkthrough on how to run this email, please check out [our
+community hour demo](https://www.youtube.com/watch?v=bLFGnoApWeU) in which we
+show how to run the various pipelines, in what order, all using an AWS stack.
+
+# 🖥 Run the example
+
+## 👣 Step-by-Step
+
```

Comment:
I'd suggest to add a header "Infrastructure Setup" here under which you group the three sections for AWS, Azure, GCP as subheaders and also add a short comment in the beginning clarifying again that you can use any of the three cloud providers. 

(When I first saw this it felt like you need to do all three setups since "Step-by-Step" sounds to me like you need to follow the subsequent steps one-by-one.)